cloudwatch-alarms.tf

```diff
@@ -78,3 +78,26 @@ resource "aws_cloudwatch_metric_alarm" "high_cpu_usage" {
   }
 
 }
+
+resource "aws_cloudwatch_metric_alarm" "ecs_running_tasks" {
+  count = length(var.alarm_sns_topics) > 0 && var.alarm_ecs_running_tasks_threshold != 0 ? 1 : 0
```

Comment:
alarm_ecs_running_tasks_threshold is = 1 by default.
is it right ?  
so, why not `var.alarm_ecs_running_tasks_threshold > 1` for this condition

Reply:
Please check the variable description for more information. 
https://github.com/DNXLabs/terraform-aws-ecs-app/blob/6785b0329b7481ce4e5a59e752da24cd83597acc/_variables.tf#L482

Reply:
fair,  alarm_ecs_running_tasks_threshold should be strongly typed for number 
 `type        = number`
and the condition
`count = length(var.alarm_sns_topics) > 0 && var.alarm_ecs_running_tasks_threshold > 0 ? 1 : 0 `

Reply:
Well spotted, thanks. The variable type has been updated.
Regarding the condition, it follows the same module pattern.

Reply:
bad pattern 🤣  this is a number, not a string,  for example, if someone defines alarm_ecs_running_tasks_threshold with "00". it will break the logic as  `00 != 0` = `true`